extensions/go-package-search/src/index.tsx

```diff
@@ -0,0 +1,164 @@
+import { ActionPanel, Action, List, showToast, Toast, Icon } from "@raycast/api";
+import { useState, useEffect, useRef, useCallback } from "react";
+import fetch, { AbortError } from "node-fetch";
+import { parse } from "node-html-parser";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search Go packages..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.name} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      icon={searchResult.search ? Icon.MagnifyingGlass : "list-icon.png"}
+      key={searchResult.fullPath}
+      title={searchResult.name}
+      subtitle={searchResult.description}
+      accessoryTitle={searchResult.fullPath}
+      accessoryIcon={searchResult.standardLibrary ? Icon.Pin : undefined}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.OpenInBrowser title="Open in Browser" url={searchResult.url} />
```

Comment:
```suggestion
            <Action.OpenInBrowser url={searchResult.url} />
```
It could be removed since it's the default title